src/LodeRunner.Core/Extensions/LoadTestConfigExtensions.cs

```diff
@@ -20,32 +20,27 @@ public static class LoadTestConfigExtensions
         /// If payloadPropertiesChanged list exists, only the listed properties are converted.
         /// </summary>
```

Comment:
nit: remove reference to payloadProperteisChanged list in summary comment :)

Reply:
good eye., removed